exercises/grep/tests/grep.rs

```diff
@@ -0,0 +1,483 @@
+extern crate grep;
+
+use grep::grep;
+
+use std::{
+    fs, sync::{Once, ONCE_INIT},
+};
+
+static ILIAD_CONTENT: &'static str = "Achilles sing, O Goddess! Peleus' son;
+His wrath pernicious, who ten thousand woes
+Caused to Achaia's host, sent many a soul
+Illustrious into Ades premature,
+And Heroes gave (so stood the will of Jove)
+To dogs and to all ravening fowls a prey,
+When fierce dispute had separated once
+The noble Chief Achilles from the son
+Of Atreus, Agamemnon, King of men.
+";
+
+static MIDSUMMER_NIGHT_CONTENT: &'static str = "I do entreat your grace to pardon me.
+I know not by what power I am made bold,
+Nor how it may concern my modesty,
+In such a presence here to plead my thoughts;
+But I beseech your grace that I may know
+The worst that may befall me in this case,
+If I refuse to wed Demetrius.
+";
+
+static PARADISE_LOST_CONTENT: &'static str = "Of Mans First Disobedience, and the Fruit
+Of that Forbidden Tree, whose mortal tast
+Brought Death into the World, and all our woe,
+With loss of Eden, till one greater Man
+Restore us, and regain the blissful Seat,
+Sing Heav'nly Muse, that on the secret top
+Of Oreb, or of Sinai, didst inspire
+That Shepherd, who first taught the chosen Seed
+";
+
+static BLOK_CONTENT: &'static str = "Белой ночью месяц красный
```

Comment:
The text is one of the many poems by the Russian poet of the Silver Age of the Russian Literature [Alexander Blok](https://en.wikipedia.org/wiki/Alexander_Blok). The poet lived and wrote in the beginning of 20th century even before Soviet period, and this particular poem([In The White Night](https://lyricstranslate.com/ru/белой-ночью-месяц-красный-white-night-crimson-crescent.html)) is about poet's fascination with the night scenery, with no veiled political or cultural propaganda, if this is your concern.
Should this be enough?

Reply:
Yes, that's great, thanks! Please add at least the link to the translation as a comment in the code; other students are also likely to be curious about the poem.

The other content blocks are named for the work, not the author, so this should probably be named `IN_THE_WHITE_NIGHT_CONTENT` instead of `BLOK_CONTENT`.